### ISSUE TITLE
decidim/decidim version is not generated properly

### ISSUE THREAD

**Issue by mrcasals** [decidim/docker#85]
Yesterday I manually triggered the build for the images for the 0.24.0.rc1 version in Decidim. Here's the action logs:

- Docker Hub: https://github.com/decidim/docker/runs/2030472562
- GitHub Registry: https://github.com/decidim/docker/runs/2030472838

But I'm getting errors of a mismatch in the Ruby versions when trying to run the images:

```
$ docker run --rm -it decidim/decidim-generator:latest ruby --version
...
ruby 2.7.1p83 (2020-03-31 revision a0c7c23c9c) [x86_64-linux]
$ docker run --rm -it decidim/decidim:latest ruby --version
...
ruby 2.6.6p146 (2020-03-31 revision 67876) [x86_64-linux]
$ docker run --rm -it decidim/decidim:0.24.0.rc1 ruby --version
...
ruby 2.6.6p146 (2020-03-31 revision 67876) [x86_64-linux]
```

The workflows seem to first build and push the `decidim-generator` image, although for some reason it lacks the 0.24.0.rc1 tag. This correctly sets the Ruby version to 2.7.1. Then, on top of that image, it builds the `decidim/decidim` one, but it's using an older version of the `decidim/decidim-generator` which still uses Ruby 2.6.6.

We should get these versions correctly.

**Comment by mrcasals** [comment]
@oliverbarnes 👀

**Comment by mrcasals** [comment]
I think I found the issue: I started the Docker registry action first, and after a while I started the GitHub Registry one. The main image uses by default the GitHub Registry version as a `FROM` image, and at the time of building it the GitHub Registry version of the `decidim/decidim-generator` image was not ready.

Proof of that is that the GH Registry version of the image works as expected:

```
$ docker run --rm -it ghcr.io/decidim/decidim:0.24.0.rc1 ruby --version
ruby 2.7.1p83 (2020-03-31 revision a0c7c23c9c) [x86_64-linux]
```

I'm working on a fix!
